internal/services/loadbalancer/loadbalancer_resource.go

```diff
@@ -53,6 +53,16 @@ func loadBalancer() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"sku": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(network.LoadBalancerSkuNameBasic),
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(network.LoadBalancerSkuNameBasic),
+				}, true),
```

Comment:
we should never ignore case unless requires
```suggestion
				}, false),
```